index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Talha Zuberi's Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>    
+    
+    <header>
+        <nav class="app-header">
+        <p class="logo-placeholder">Welcome!</p>
+                <img src="menu-button.svg" alt="Navigation">
+        </nav>
+    </header>
+    <div class="app-wrapper">
+        
+        <section class="welcome-line">    
+            <h2>I am Talha.<br>Glad to see you!</h2>
```

Comment:
- [ ] Kindly use the `h1` tag here, that will make your code more semantic and structured